app/models/project.rb

```diff
@@ -1,3 +1,32 @@
+require 'csv'
+
 class Project < ApplicationRecord
 
+  def self.to_csv(ids = [])
+
+    csv = ''
+
+    project_columns = ["Number", "Project Title", "Donor(s)", "Status", "Start Date", "End Date",
```

Comment:
Project columns can be retrieved using the Project model like this `Project.new.attributes.keys` and then formatted using some ruby functions like `gsub` to replace the underscore and `capitilize` to set the first character of an attribute as a capital letter.
In this way we don't need to have the attributes written manually and we rely directly on the projects attributes themselves. 
Also remember that we don't have `Number` anymore but it's just the ID. The method described above will take care of this anyway